src/write.c

```diff
@@ -3073,6 +3073,18 @@ avifResult avifEncoderFinish(avifEncoder * encoder, avifRWData * output)
             AVIF_CHECKRES(avifRWStreamWriteChars(&s, "MA1A", 4));          // ... compatible_brands[]
         }
     }
+#if defined(AVIF_ENABLE_EXPERIMENTAL_GAIN_MAP)
+    for (uint32_t itemIndex = 0; itemIndex < encoder->data->items.count; ++itemIndex) {
+        if (!memcmp(encoder->data->items.item[itemIndex].type, "tmap", 4)) {
+            // ISO/IEC 23008-12:2024/AMD 1:2024(E)
+            // This brand enables file players to identify and decode HEIF files containing tone-map derived image
+            // items. When present, this brand shall be among the brands included in the compatible_brands
+            // array of the FileTypeBox.
```

Comment:
Maryla: Thank you for the fix in src/write.c and the test files. I assume you will also fix src/read.c (the subject of https://github.com/AOMediaCodec/libavif/issues/2385)?

Reply:
Yes that will be in another PR.